### DISCUSSION TITLE
Easiest way to Add/Test a register (Heat/Hot water)

### DISCUSSION THREAD

**Discussion by andrasj** [yozik04/nibe#85] [Q&A] [answered]
I own a Nibe F1255PC, and I'm using a esphome/Lillycan board to monitor some values.
I kept looking for some indication when the pump is working for the integrated boiler instead of regular heating, but I could not find an entity (the app does show an additional icon when the pump is heating the boiler).
Digging through some documentation/manuals and searching the internet, I found a document with multiple registers for various nibe heat-pumps. In this document they mentioned: "40314 ; QN10 Need ; 0=Heating, 1=Hot water ; u8"
but this register was not marked as available for the F1255PC. However the technical documentation also mentioned QN10 as the valve responsible for switching between climateSystem/boiler.
I think it's worth trying to read this register (except if somebody else can point me to the correct register).
I'm pretty new to homeassistant, so my experience is not that huge, but I'm willing to dig deeper.
But what would be the easiest way to test? Is it possible to run some commands directly from homeassistant? (HAOS on RPi3)

**Comment by yozik04** [comment, marked as answer]
prio-43086 is the right register.
